amy/extcomments/tests.py

```diff
@@ -1 +1,39 @@
-# Create your tests here.
+from django.test import TestCase
+import django_comments
+
+from workshops.models import Organization, Person
+
+
+class TestEmailFieldRequiredness(TestCase):
+    def test_email_field_requiredness(self):
+        """Regression test for #1944.
+
+        Previously a user without email address would not be able to add a comment."""
+        # Arrange
+        person = Person.objects.create(
+            personal="Ron",
+            family="Weasley",
+            username="rw",
+            is_active=True,
+            email="",
+        )
+        person.set_password("testrwpassword")
+        self.client.login(username="rw", password="testrwpassword")
```

Comment:
@pbanaszkiewicz can you assert that self.client.login returns True? 

I have a feeling it might not, and it's essential for this test.

Reply:
@lauryndbrown you were right, it didn't log the user in. No wonder we're using `force_login` so much in tests.

I'm reworking the tests, and I'll resubmit this for your review afterwards.